cms/src/components/content/EditContent.vue

```diff
@@ -397,6 +397,61 @@ const liveUrl = computed(() => {
 });
 
 const ensureRedirect = () => window.open(liveUrl.value, "_blank");
+
+const showDuplicateModal = ref(false);
+
+const duplicate = async () => {
+    showDuplicateModal.value = false;
+
+    if (!editableParent.value) return;
+
+    // Handle new data for duplicated document and keep old data
+    const clonedParent = _.cloneDeep(editableParent.value);
+    const newParent = { ...clonedParent, _id: db.uuid() };
+
+    // Remove Original Image
+    if (newParent.imageData) {
+        if (newParent.imageData.fileCollections) {
+            newParent.imageData.fileCollections = [];
+        }
+    }
+
+    // Duplicate all content documents and make them unique
+    const duplicatedContent = editableContent.value.map((c) => {
+        const newContent = _.cloneDeep(c);
+        newContent._id = db.uuid();
+        newContent.updatedTimeUtc = Date.now();
+        newContent.title += " (Copy)";
+        newContent.slug += "-copy";
+        newContent.parentId = newParent._id;
+        newContent.status = PublishStatus.Draft;
+
+        return newContent;
+    });
+
+    editableParent.value = newParent;
+    editableContent.value = duplicatedContent;
+
+    const isTestEnviroment = import.meta.env.MODE === "test";
+    if (!isTestEnviroment) {
```

Comment:
Add a comment to why we need to bypass the test environment.
Not needed for a separate const for just one comparison?
```suggestion
    if (import.meta.env.MODE !== "test") {
```